v0.3/tutorials/quickstart 2 button stopwatch.eve

```diff
@@ -0,0 +1,252 @@
+
+## Drawing the buttons
+
+This time there are two buttons instead of one.
+
+````
+```
+search
+  [#app page: "Clock"]
+  view = [#qs-contents]
+
+bind
+  view.children += [#ui/div sort: 2 children:
+    [#ui/div #qs-stopwatch-btn #start-stop-btn text: "Start/Stop"]
+    [#ui/div #qs-stopwatch-btn #reset-btn text: "Reset"]]
+```
+````
+
+Now we have two buttons - one for start/stop and another for reset.
+
+## Pressing the buttons
+
+This is where things get a little bit tricky. We're going to keep the same three pieces of button logic from last time, although additional blocks will be required to give us two button functionality.
+
+### Starting from a reset
+
+````
+```
+search
+  [#time timestamp]
+  race-timer = [#race-timer #stopped #reset]
+  [#html/event/click element: [#start-stop-btn]]
+
+commit
+  [#start-time timestamp]
+  race-timer -= #stopped
+  race-timer -= #reset
+  race-timer += #running
+```
+````
+
+This is exactly what the start button did in the last example.
+
+### Stopping the first time
+
+````
+```
+search
+  [#time timestamp]
+  not([#stop-time])
+  race-timer = [#race-timer #running]
+  [#html/event/click element: [#start-stop-btn]]
+
+commit
+  [#stop-time timestamp]
+  race-timer += #stopped
+  race-timer -= #running
+```
+````
+
+This is close to what the stop button did in the last example, except it checks to make sure there's no existing `#stop-time`. This means it's the first time the stop button has been pressed since a reset, and `#stop-time` needs to be created.
+
+### Pressing Reset
+
+````
+```
+search
+  race = [#race-timer #stopped]
+  start = [#start-time]
+  stop = [#stop-time]
+  [#html/event/click element: [#reset-btn]]
+
+commit
+  race := none
+  start := none
+  stop := none
+  [#race-timer #stopped #reset]
+```
+````
+
+The reset button's only difference from last time is that it's discrete from the start/stop button, and as such this blocks looks specifically for the `#reset-btn` to be pushed.
+
+### Resuming from a stop
+
+````
+```
+search
+  [#time timestamp]
+  start = [#start-time]
+  stop = [#stop-time]
+  race-timer = [#race-timer #stopped not(#reset)]
+  [#html/event/click element: [#start-stop-btn]]
+
+commit
+  start.timestamp := start.timestamp + (timestamp - stop.timestamp)
+  race-timer -= #stopped
+  race-timer += #running
+```
+````
+
+This is where things get a little tricky. This new block checks for the start button to be pressed while the stopwatch is stopped and checks for existing `#start-time` and `#stop-time` records, which indicate that we're resuming from a stop and not a reset. It then changes the `#race-timer` from `#stopped` back to `#running` and gives `#start-time` a new timestamp, which is the sum of its original value and the amount of time the stopwatch was stopped (the difference between the timestamp upon resuming and the timestamp when it was stopped). This lets the hands pick up exactly where they left off.
+
+### Subsequent stops
+
+````
+```
```

Comment:
While yes is technically an answer I was hoping you might expound for my benefit